2021-04-21T18:34:26.3484543+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:34:26.3529677+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:34:26.3542487+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:34:26.3553638+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:34:26.3564986+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:34:27.3761889+02:00 0HM84SDA4HAN3:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:34:37.4519421+02:00 0HM84SDA4HAN3:0000000D [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'ShoppingCart.Application.Interfaces.ITasksService' while attempting to activate 'WebApplication1.Controllers.TasksController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-21T18:35:10.8878719+02:00 0HM84SDA4HAN3:0000000F [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'ShoppingCart.Application.Interfaces.ITasksService' while attempting to activate 'WebApplication1.Controllers.TasksController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-21T18:35:18.0547577+02:00 0HM84SDA4HAN3:00000011 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

WebApplication1.Controllers.TasksController.Create (WebApplication1)
WebApplication1.Controllers.TasksController.Create (WebApplication1)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HstsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
2021-04-21T18:36:25.9026878+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:36:25.9072735+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:36:25.9086743+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:36:25.9099660+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:36:25.9111398+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:36:26.7895179+02:00 0HM84SEDO0GK7:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:36:39.1246899+02:00 0HM84SEDO0GK7:00000005 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'ShoppingCart.Application.Interfaces.ITasksService' while attempting to activate 'WebApplication1.Controllers.TasksController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-21T18:38:59.5685556+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:38:59.5721214+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:38:59.5732612+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:38:59.5742470+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:38:59.5752381+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:39:01.1937846+02:00 0HM84SFRHRF31:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:39:06.7286171+02:00 0HM84SFRHRF31:00000003 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'ShoppingCart.Application.Interfaces.ITasksService' while attempting to activate 'WebApplication1.Controllers.TasksController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-21T18:39:14.1529757+02:00 0HM84SFRHRF31:00000005 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'ShoppingCart.Application.Interfaces.ITasksService' while attempting to activate 'WebApplication1.Controllers.TasksController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-21T18:40:03.3573568+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:40:03.3611231+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:40:03.3626209+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:40:03.3636811+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:40:03.3648206+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:40:03.9185015+02:00 0HM84SGEEJQ5T:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:42:10.1255393+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:42:10.1292157+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:42:10.1306150+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:42:10.1318135+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:42:10.1329989+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:42:11.0267229+02:00 0HM84SHKBE63A:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:42:18.0452202+02:00 0HM84SHKBE63A:00000005 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

WebApplication1.Controllers.TasksController.Create (WebApplication1)
WebApplication1.Controllers.TasksController.Create (WebApplication1)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HstsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
2021-04-21T18:42:19.0272012+02:00 0HM84SHKBE63A:00000007 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

WebApplication1.Controllers.TasksController.Create (WebApplication1)
WebApplication1.Controllers.TasksController.Create (WebApplication1)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.HttpsPolicy.HstsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.Invoke(HttpContext context)
2021-04-21T18:45:25.3398851+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:45:25.3453350+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:45:25.3474653+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:45:25.3490647+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:45:25.3511266+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:45:26.1607996+02:00 0HM84SJEF8B06:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:54:05.9037126+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:54:05.9088992+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:54:05.9103156+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:54:05.9112450+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:54:05.9121180+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:54:06.9014331+02:00 0HM84SO9KE9UI:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:56:22.7402458+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:56:22.7442106+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:56:22.7456332+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:56:22.7466951+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:56:22.7477658+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:56:23.6481679+02:00 0HM84SPIBKNF8:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T18:59:32.1491009+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T18:59:32.1526328+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T18:59:32.1539887+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T18:59:32.1551149+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T18:59:32.1563221+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T18:59:33.1403799+02:00 0HM84SRAR67T9:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T19:01:05.6884274+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-21T19:01:05.6923736+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-21T19:01:05.6939057+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-21T19:01:05.6958918+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-21T19:01:05.6974606+02:00  [INF] Content root path: "E:\Program Files\VS REPOS\AppSecurityAssignment\WebApplication1" (b5d60022)
2021-04-21T19:01:06.5790739+02:00 0HM84SS6ORQHJ:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-21T19:09:23.2410042+02:00 0HM84SS6ORQHJ:00000011 [INF] User logged in. (fdce0db4)
2021-04-21T19:09:23.2930688+02:00 0HM84SS6ORQHJ:00000013 [INF] Home Index accessed (f306b1a0)
